roles/observe_chargeback/README.md

```diff
@@ -0,0 +1,59 @@
+Role Name
+=========
+The **`observe_chargeback`** role is designed to test the **RHOSO Cloudkitty** feature. These tests are specific to the Cloudkitty feature. Tests that are not specific to this feature (e.g., standard OpenStack deployment validation, basic networking) should be added to a common role.
+
+Requirements
+------------
+The following resources are required in the OpenStack cloud
+* network called public
+* network called private
+* security group called basic with ssh and tcp enabled to the VMs
+
+
+It relies on the following being available on the target or control host:
+
+* This role requires **Ansible 2.9** or newer.
+* The **OpenStack CLI client** must be installed and configured with administrative credentials (e.g., source the `openrc` file).
+* Required Python libraries for the `openstack` CLI (e.g., `python3-openstackclient`).
+* Connectivity to the OpenStack API endpoint.
+
+Role Variables
+--------------
+The role uses a few primary variables to control the testing environment and execution.
+
+| Variable | Default Value | Description |
+| `openstack_cmd` | `openstack` | The command used to execute OpenStack CLI calls. This can be customized if the binary is not in the standard PATH. |
+
+Dependencies
+------------
+This role has no direct hard dependencies on other Ansible roles.
+
+However, it is expected to be run **after** a successful deployment and configuration of the following components:
+
+* **OpenStack:** A functional OpenStack cloud (RHOSO) environment.
+* **logging:** Logging  service must be installed, configured, and running for Openstack
+* **Cloudkitty:** The Cloudkitty service must be installed, configured, and running.
+
+Example Playbook
+----------------
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too.
+
+Each tasks/playbook.yml should be called independently via "ansible.builtin.import_role" with appropriate vars passed:
+
+- name: "name of ansible playbook" 
+  hosts: computes
```

Comment:
The observe_chargeback role is not run on computes, change this to `controller` or `localhost` instead